pkg/controllers/edgeconnect/controller.go

```diff
@@ -82,26 +82,42 @@ func (controller *Controller) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (controller *Controller) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
-	log.Info("reconciling EdgeConnect", "name", request.Name, "namespace", request.Namespace)
+	llog := log.WithValues("namespace", request.Namespace, "name", request.Name)
```

Comment:
Nitpick: 
I kinda dislike the `llog` naming can't we find something better?

My suggestions: `_log`, `vlog`

Reply:
`llog` is short for `locallog`, but I'm not married to the name

Reply:
fixed in e3699102